### ISSUE TITLE
Webcam cam usb frame rate setting not respected. 

### ISSUE THREAD

**Issue by dener3id** [ispysoftware/iSpy#254]
I use two versions of the ps3 itoy webcam and I want it to encode the videos at 1-5 fps and it's won't go below 10 (even if the camera's not able to lower if fps) why not combine or just pick one frame to send to the recording? Yeah I can maybe write bash script to open my recordings folders and encode with ffmpeg and do that it should always respect my frame rate setting. it's saving at about 10 fps (8.something as reported in vlc and confirmed by watch time stamps for speed change) even thought both cameras I've now set to 1 fps (had one at 5 just to test having multiple fps's.

**Comment by ispysoftware** [comment]
I'm not able to reproduce this - when I record with max framerate set to 1 on record tab and mod set to encode the recording is 1fps.

**Comment by dener3id** [comment]
Yeah I know it sounds weird how do I post logs to show what's going on? I can screen shoot the ui. I don't know what else to post.

**Comment by ispysoftware** [comment]
maybe just upload your objects.xml file somewhere (from server menu - backup)
